cluster-up/cluster/external/provider.sh

```diff
@@ -5,19 +5,24 @@ function _kubectl() {
 }
 
 function prepare_config() {
-    cat >hack/config-provider-external.sh <<EOF
-docker_tag=devel
-docker_prefix=${DOCKER_PREFIX}
-manifest_docker_prefix=${DOCKER_PREFIX}
-image_pull_policy=${IMAGE_PULL_POLICY:-Always}
+    BASE_PATH=${KUBEVIRTCI_CONFIG_PATH:-$PWD}
+
+    # required for running tests within openshift ci, otherwise tests don't find the configuration
+    ln -f -s "${KUBECONFIG:-$HOME/.kube/config}" "${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig"
```

Comment:
I am not sure what you mean by that. You can set the config location on the tests binary. Is that what you try to solve here?

It probably does not work right now, but the provider is supposed to just find the config at a default location or in KUBECONFIG.

Reply:
Yes, that is exactly what I'm trying to solve here. Within openshift-ci `KUBECONFIG` is set to `/tmp/admin.kubeconfig`. Unfortunately the tests won't find the configuration unless I symlink to the target configuration.

I agree that this is a symptom, not the cause. Opened #202 for this.